tensorflow/tensorboard/backend/tensorboard_handler_test.py

```diff
@@ -25,6 +25,7 @@
 from tensorflow.python.platform import googletest
 from tensorflow.tensorboard.backend import tensorboard_handler
 
+from six.moves import xrange
```

Comment:
If we want to avoid issues later, put it above. It'll just create a merge conflict when some overzealous linter gets a hold of it.